soroban-env-host/benches/common/measure.rs

```diff
@@ -174,19 +174,26 @@ mod cpu {
     }
 }
 
-#[cfg(all(target_os = "macos", target_arch = "aarch64"))]
+#[cfg(target_os = "macos")]
+#[path = "."]
```

Comment:
What does the path attribute do here? I've read about it, but unclear how it works on a mod that has a body.

Reply:
it just overrides the default mapping of mod-with-submods-means-directory (by default rust would look for `cpu/rusagev4.rs`)